ground/src/main/res/layout/map_task_frag.xml

```diff
@@ -47,6 +47,17 @@
       android:src="@drawable/ic_gps_lock"
       android:tint="@android:color/white" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+      android:id="@+id/hintIcon"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:src="@drawable/ic_info"
+      android:layout_gravity="top|start"
+      android:visibility="gone"
+      app:backgroundTint="?attr/colorSurface"
+      app:fabSize="mini"
+      app:useCompatPadding="true" />
```

Comment:
Should this be a part of the TaskView template?

Reply:
I'm not sure that's necessary because as far as I'm aware this hint button is only going to be used on the map tasks.

Reply:
Ack

Reply:
But then where will we show the task label for photo task?